.github/workflows/release-post-merge.yml

```diff
@@ -0,0 +1,73 @@
+# GitHub workflow for createing release.
+# Trigger release branch should be merge into main
+
+name: Create Release.
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  create-release:
+    name: Create Release
+    if: (github.event.pull_request.merged == true) && (startsWith(github.event.pull_request.head.ref, 'release'))
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Print Env and Get version
+      id: tag
+      env:
+        BRANCH: ${{ github.event.pull_request.head.ref }}
```

Comment:
[nit] took me a while to understand this - I assume this is not a branch but a PR name? Does it make sense to rename `BRANCH` to `PULL_REQUEST_NAME` maybe?